app/screens/map-screen/map-screen.tsx

```diff
@@ -70,38 +99,84 @@ export const MapScreen: React.FC<Props> = ({ navigation }) => {
 
   const maps = data?.businessMapMarkers ?? []
 
+  // if getting location was denied and device's country code has been found (or defaulted)
+  // this is used to finalize the initial location shown on the Map
+  React.useEffect(() => {
+    if (countryCode && wasLocationDenied && !loading) {
+      // El Salvador gets special treatment here and zones in on El Zonte
```

Comment:
most users in ES by now are not in El Zonte, so I think we don't need this exception, we could show the whole El Salvador map imo

Reply:
removed